conf/sample.ini

```diff
@@ -1228,3 +1228,14 @@
 
 # Enable or disable loading other base map layers
 ;enable_custom_baselayers = true
+
+# Move an app plugin referenced by its id (including all its pages) to a specific navigation section 
+# Dependencies: needs the `topnav` feature to be enabled
+[navigation.app_sections]
+# The following will move an app plugin with the id of `my-app-id` under the `starred` section
+# my-app-id = starred
```

Comment:
```suggestion
# my-app-id = admin
```

Starred is a strange section to add plugin pages to as it's not really a real nav section :)

Reply:
Makes sense, thanks 👍